shared/src/test/diff/nu/WeirdUnions.mls

```diff
@@ -11,23 +11,23 @@ fun f: [Str] | [Str, Int]
 fun f: (Str | [Str, Int])
 //│ fun f: Str | [Str, Int]
 
-:pe
+:e
 fun f: Str | (Str, Int)
-//│ ╔══[PARSE ERROR] Expected '=>' or '->' after this parameter section
+//│ ╔══[ERROR] type identifier not found: ,
```

Comment:
Wow, so can `,` be a user-defined type operator right now?

Reply:
Nah because `,` will parse as a COMMA token.

```scala
type , = 1
//│ ╔══[PARSE ERROR] Expected a type name; found comma instead
//│ ║  l.4: 	type , = 1
//│ ╙──     	     ^
//│ TEST CASE FAILURE: There was an unexpected parse error
//│ type <error> = 1
```

Anyway in the future terms and types will parse the same, so this will be moot.